test/routes/article-list.test.ts

```diff
@@ -1,13 +1,20 @@
 import jsonld from 'jsonld';
 import { Next, Response } from 'koa';
+import { JsonLdObj } from 'jsonld/jsonld-spec';
 import articleList from '../../src/routes/article-list';
 import createContext from '../context';
 import runMiddleware from '../middleware';
 
 const makeRequest = async (next?: Next): Promise<Response> => {
   const context = createContext();
-
-  return runMiddleware(articleList(context.router), context, next);
+  const articles = {
+    all: (): Iterable<JsonLdObj> => [],
+    add: jest.fn(),
+    get: jest.fn(),
+    has: jest.fn(),
+  };
```

Comment:
We should create a proper `NullArticles` implementation which is empty (similar to https://github.com/libero/content-api-bundle/blob/master/src/Adapter/NullItems.php).

Reply:
I'm resolving as we have a `NullArticles` adaptor now.